usbsdmux/usb2642.py

```diff
@@ -450,3 +454,27 @@ def write_to(self, i2cAddr, data):
             raise I2cTransactionFailed(
                 "SCSI-Transaction ended with status {}. I2C-Transaction has probably failed.".format(sgio.status)
             )
+
+    def _read_register(self, reg, size):
+        scsiCommand = list_to_uint8_array(
+            [0xCF, reg, 0x00, 0x00, size, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00], 16
```

Comment:
In the rest of this file we try to avoid magic numbers like ``0xcF``. I would prefer to add a module level variable for key numbers here, too.

Reply:
I just followed the existing approach from: https://github.com/linux-automation/usbsdmux/blob/master/usbsdmux/usb2642i2c.py#L359

Reply:
In this case: Works for me. Maybe someone can pick up documentation on this later.